pkg/kyma/cmd/test/cmd.go

```diff
@@ -0,0 +1,155 @@
+package test
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/kyma-incubator/kymactl/internal"
+	kyma_helm "github.com/kyma-incubator/kymactl/internal/helm"
+	"github.com/kyma-incubator/kymactl/internal/step"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/helm/pkg/helm"
+	"k8s.io/helm/pkg/helm/environment"
+	helm_release "k8s.io/helm/pkg/proto/hapi/release"
+)
+
+type Options struct {
+	step.Factory
+	skip []string
+	verbose bool
```

Comment:
Do you have an idea how to have it as a global parameter? The verbose flag will be important for all commands and I don't want to duplicate it's description in every command.

Reply:
I moved the Verbose flag to KymaOptions and made TestOptions extend it. I also moved step.Factory there so every command can have it available

Reply:
works like a charm. I rolled it out to all commands